### PR TITLE
Chore/updates

### PR DESCRIPTION
**Pull request by SimonSchick**
Tested locally, site looks good.

This should help a little bit with maintainability.

Next step would be to ditch the `toJSON` call from raml and use their actual calls which should make things 99% less annoying.

### CODE REVIEW

File: build/misc.ts [hunk 1]
```diff
@@ -43,17 +39,17 @@ function wrappedRequire (filePath) {
  * @return {Object}
  */
 function getLocals () {
-    const restDoc = readJSONFile(path.join(__dirname, '/tmp/raml-doc.json'));
+    // It's complicated.
```
Comment (connor4312) [inline build/misc.ts]: lol

File: build/util.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/**
+ * Creates a new object with all values from the passed object ordered by keys.
+ */
+export function orderObject<T, K extends keyof T>(obj: T): { [P in K]: T[P]; } {
```
Comment (connor4312) [inline build/util.ts]: K is extraneous here and is looser than `keyof T`. This works:

```ts
function orderObject<T>(obj: T): { [P in keyof T]: T[P]; } {
    const ret: { [P in keyof T]: T[P]; } = <any>{};
    const orderedKeys = <(keyof T)[]>Object.keys(obj).sort();
    for (const key of orderedKeys) {
        ret[key] = obj[key];
    }
    return ret;
}
```

**Comment by ProbablePrime** [comment]
@SimonSchick Noticed you're not on OOO From tommorrow(Tuesday). When/if you get this building we can merge it.

I'd suggest rebasing a bit as i've made some changes to the developer site as indicated by the conflicts

**Comment by ProbablePrime** [comment]
Closing due to lack of interest @SimonSchick If you're interested please re-open.

**Pull request closed without merging**